<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.238636, x2:1, y2:1, stop:0 rgba(49, 116, 96, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Vemana2000&quot;;
background-color: rgb(136, 178, 180);</string>
       </property>
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="MessageLineEdit">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Vemana2000&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SendAllMessage">
       <property name="font">
        <font>
         <family>Vemana2000</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Vemana2000&quot;;
background-color: rgb(136, 178, 180);</string>
       </property>
       <property name="text">
        <string>Send all:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SendPrivateMessage">
       <property name="font">
        <font>
         <family>Vemana2000</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Vemana2000&quot;;
background-color: rgb(136, 178, 180);</string>
       </property>
       <property name="text">
        <string>Send private:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>45</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="CommonChatBrows">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Vemana2000&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="PrivateChatBrows">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Vemana2000&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_menu">
    <property name="title">
     <string>Main menu</string>
    </property>
    <addaction name="actionOpen_another_client"/>
    <addaction name="separator"/>
    <addaction name="actionClose_this_client"/>
   </widget>
   <addaction name="menuMain_menu"/>
  </widget>
  <action name="actionOpen_another_client">
   <property name="text">
    <string>Open another client</string>
   </property>
  </action>
  <action name="actionClose_this_client">
   <property name="text">
    <string>Close this client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
